tox.ini

```diff
@@ -13,7 +13,9 @@ install_command =
 deps =
   -r{toxinidir}/requirements.txt
   -r{toxinidir}/test-requirements.txt
-commands = stestr run {posargs}
+commands =
+  mypy --config-file mypy.ini git_machete/cmd.py
```

Comment:
Actually... can you already make mypy check the entire package/directory, instead of just a single file? 🤔